data/preset/buffet.yaml

```diff
@@ -118,7 +121,9 @@ defines:
                             - 0
                             - 1
                     selection:
-                      type: Selection.AllHighlights
+                      type: Selection.FeaturedFirst
```

Comment:
Yes, I see that too. It seems like it's querying 4 models, then 3, then 2, then 1 model, but only adding one each time. And it's not happening on master.